shared/utils/galaxy_ansible_role_meta_template.yml

```diff
@@ -0,0 +1,17 @@
+galaxy_info:
+  author: OpenSCAP
+  description: @DESCRIPTION@
+  company: Red Hat, Inc.
+
+  license: BSD
```

Comment:
?

Reply:
That's the same license I used on that first PR and nobody complained. Which one should we use, GPL?